### ISSUE TITLE
ko.properties is old

### ISSUE THREAD

**Issue by PC28K** [klyte45/TransportLinesManager#100]
I don't think the change was applied to the actual TransportLinesManager.dll file. please check.

**Comment by klyte45** [comment]
This is the current version of ko.properties sent by you and merged about 26 days ago:
https://github.com/klyte45/TransportLinesManager/blob/master/UI/i18n/ko.properties

Is it wrong?

**Comment by PC28K** [comment]
that's right, but I decompiled it because it didn't seem to be applied in the game, but there was an older version of the file. so i thought something was wrong.

**Comment by klyte45** [comment]
Double check the dll file version to ensure it's the current dll file. The stable version is 14.0.0.3

**Comment by PC28K** [comment]
It's school soon, so I'll be able to check it out in about 9 hours. but I think it's already the latest version. Because I saw the change history window when I loaded the game, and I downloaded it from the workshop yesterday.

**Comment by PC28K** [comment]
![l](https://user-images.githubusercontent.com/41824434/119790381-d80b5200-bf0e-11eb-82a0-57f900677b2d.PNG)

It is certainly different from 
 https://github.com/klyte45/TransportLinesManager/blob/master/UI/i18n/ko.properties

**Comment by klyte45** [comment]
Just thinking from where it got this file...
Anyway, it must be reviewed next patch

**Comment by klyte45** [comment]
Found the issue here
Will be fixed in next patch, sorry for delaying it!

**Comment by PC28K** [comment]
thank you~
